action.yaml

```diff
@@ -0,0 +1,66 @@
+# action.yml
+name: "codacy-analysis-cli"
+description: "Execute Codacy code analysis using your remote Codacy configuration"
+branding:
+  icon: 'check'
+  color: 'gray-dark'
+inputs:
+  project-token:
+    required: true
+  directory:
+    required: false
+    default: "`pwd`"
+  tool-timeout:
+    required: false
+    default: "15minutes"
+  output:
+    required: false
+  format:
+    required: false
+    default: "text"
+  upload:
+    required: false
+    default: false
+  parallel:
+    required: false
+    default: 2
+  max-allowed-issues:
+    required: false
+    default: 0
+  fail-if-incomplete:
+    required: false
+    default: false
+  allow-network:
+    required: false
+    default: false
+  force-file-permissions:
+    required: false
+    default: false
+runs:
+  using: "docker"
+  image: "docker://codacy/codacy-analysis-cli:stable"
+  args:
+    - "analyse"
+    - "--project-token"
+    - "${{ inputs.project-token }}"
+    - "--directory"
+    - "${{ inputs.directory }}"
+    - "--tool-timeout"
+    - "${{ inputs.tool-timeout }}"
+    - "--output"
+    - "${{ inputs.output }}"
```

Comment:
Since GH actions do not support YAML conditionals, and the `output` flag is not mandatory I need to see what happens when this is empty.

Reply:
The CLI fails to run if this is empty. I think I'll create some sort of default file and always run the CLI with `--verbose`.

Reply:
This was solved by not using `docker` mode in GH actions to run the CLI. By using the `composite` mode we can invoke bash and use the conditionals from there.